ckanext/privatedatasets/parsers/fiware.py

```diff
@@ -61,4 +60,4 @@ def parse_notification(self, request_data):
             else:
                 raise tk.ValidationError({'message': 'Invalid resource format'})
 
-        return {'users_datasets': [{'user': user_name, 'datasets': datasets}]}
+        return {'users_datasets': [{'user': user_name, 'datasets': datasets}]}
```

Comment:
You have removed the line brake at the end of the file. It is not very important but PEP8 will complain